### ISSUE TITLE
Turn off zoom to width

### ISSUE THREAD

**Issue by ntgxtreme** [kenmoore/sketch-artboard#2]
Hello,

Is it possible to disable zooming to artboard width? Thanks!

**Comment by blcarson** [comment]
Haven't found a way to do this myself. Resorted to resizing my window so that on artboard switch it ended up zooming to 100%

**Comment by dropcapdreams** [comment]
@ntgxtreme @blcarson I had the same issue, when going to the next Artboard it would "Zoom to Fit" the Artboard. I was able to make a small edit to the code so if your viewing at 100% and go to the next Artboard, it would stay at 100% (no zooming to fit). The only issue is if your zoomed in and your Artboard is larger then the view port width, then it reverts back to "Zoom to Fit". This is usually never the case for me as I usually switch Artboards at 100% view or even less sometimes.

I changed line 93 from:
var newZoomValue = viewportWidth / newWidth;
to
var newZoomValue = [doc zoomValue];

Basically keeps the zoom view as is, instead of assigning a new one. I'm not sure if this is @kenmoore intention, but I find it makes this plugin 1000x more useful.

**Comment by neuedeutsche** [comment]
mhm i made the change as suggested by @dropcapdreams but that makes the artboard stick to the top of the viewport, also not ideal.

basicly we dont want it to deal with zooming at all just switch to the next artboard and do whatever cmd + 3 would do in one step am i right?

**Comment by dropcapdreams** [comment]
@dergraph 

I made another edit but forgot to mention it here sorry!

You need to change line 93 AND 94 from:

var newZoomValue = viewportWidth / newWidth;
var newY = Math.max([nextArtboardRect y], [nextArtboardRect y] - ([currentArtboardRect y] + scrollOrigin.y / zoomValue));

TO

var newZoomValue = [doc zoomValue];
var newY = Math.max([nextArtboardRect y] - ([currentArtboardRect y] + scrollOrigin.y / zoomValue));

After doing this, this will not only prevent the document to zoom to 100% width but also keep the exact view position. I use it all the time, love it.

**Comment by blcarson** [comment]
@dropcapdreams that worked perfect, thank you!

**Comment by dropcapdreams** [comment]
@blcarson Dont' know if you know, but the latest version of Sketch now has this functionality built in and it works great. On the mac it's Function + arrow (left or right)

**Comment by blcarson** [comment]
Did not - thanks for the heads up!
